libvmi/driver/xen/xen.c

```diff
@@ -835,6 +836,26 @@ xen_init_events(
 }
 
 status_t
+xen_init_altp2m (
```

Comment:
So since there is no separate flag the user sends to specifically activate the altp2m parts this function doesn't need to return a status_t and can just be void. You could also make it into a static inline and move it to altp2m_private.h